doc/packet_struct_notation.md

```diff
@@ -7,6 +8,11 @@
 //===== Description: =========================================
 //= Explanation how packets are and should be documented.
 //============================================================
+-->
+
+# Packet Structure Notation
+> Author: Ai4rei  
+> Last updated: 08/10/2012
```

Comment:
```suggestion
```
We have versioning at the top + git history. I would remove this kind of information.